rosplane/scripts/rosplane_gcs_launch.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Function to print help instructions
+print_help() {
+    usage
+    echo
+    echo "Options:"
+    echo "  -h                Show this help message and exit"
+    echo "  -s                Run as simulation of ROSflight"
+    echo "  -t                Run tuning version of ROSflight"
+    echo "  -r                Run the script with simulated transmitter"
+    echo "  -a                Aircraft option to pass to launch files"
+    echo "  aircraft          The aricraft parameter to be used in launch files, default is anaconda"
+    echo "  -b                Start a ROS bag"
+    ehco "  bag_name          Name of bag (optional)"
+    echo "  -o                Run the tmux session online (on remote host)"
+    echo "  user@host         Username and address of remote"
+    echo "  -d                Indicate that you should run a docker container of the given name (uses compose, and compose file should be in workspce directory)"
+    echo "  container       Name of container"
+    echo "  path/to/workspace Path to workspace"
+}
+
+usage() {
+    echo "Usage: $0 [-h] [-s] [-t] [-r] [-a aircraft] [-b [bag_name]] [-o user@host] [-d docker_container_name] path/to/ROSflight/workspace"
+}
+
+sim=false
+tuning=false
+rc_sim=false
+aircraft='anaconda'
+online=false
+docker=false
+bag=false
+
+# Parse options using getopts
+while getopts ":hstra:b:o:d:" opt; do
+    case $opt in
+        h)
+            
+            print_help
+            exit 0
+            ;;
+        s)
+            sim=true
+            ;;
+        t)
+            tuning=true
+            ;;
+        r)
+            rc_sim=true
+            if ! $sim; then
+              echo "Cannot run simulated transitter, while not running a simulation."
+              exit 1 
+            fi
+            ;;
+        a)
+            aircraft=$OPTARG
+            ;;
+        b)
+            bag=true
+            bag_name=$OPTARG
+            ;;
+        o)
+            online=true
+            user_host=$OPTARG
+            if $sim; then
+              echo "Cannot run simulat on a remote."
+              exit 1 
+            fi
+            ;;
+        d)
+            docker=true
+            container=$OPTARG
+            if ! $online; then
+              echo "Script not configured for Docker not on remote."
```

Comment:
Script not configured, docker not on remote? I'm not certain what this is saying.

Reply:
Ya, I need to rethink these. I was planning on asking you about it actually!

Reply:
So essentially it means that the script won't try and run docker on your local machine if you put the options for the script to do so.